style.css

```diff
@@ -133,3 +134,230 @@ nav img {
   position: relative;
   left: -3vh;
 }
+
+/* Works */
+
+.card {
+  min-height: 552px;
+  width: 90%;
+  margin: auto;
+  display: flex;
+  justify-content: center;
+  background-color: white;
+  border: 1px solid #dfe1e6;
+  margin-top: 180px;
+  border-radius: 16px;
+}
+
+.card > div {
+  font-family: "Poppins", sans-serif;
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+  margin: 5%;
+  width: 100%;
+}
+
+.card img {
+  border-radius: 8px;
+}
+
+.card h2 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 200%;
+  line-height: 44px;
+}
+
+.details {
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+}
+
+.details h3 {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 80%;
+  line-height: 16px;
+  position: relative;
+  top: 4px;
+}
+
+.details li {
+  display: inline;
+  margin-right: 10px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 80%;
+  line-height: 16px;
+}
+
+.card p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 95%;
+  line-height: 160%;
+  color: #344563;
+}
+
+.tech {
+  display: inline-grid;
+  display: -moz-inline-grid;
+  grid-template-columns: 0.5fr 0.5fr 2fr;
+  gap: 8px;
+}
+
+.tech li {
+  font-weight: 500;
+  font-size: 77%;
+  line-height: 133%;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  text-align: center;
+  padding: 4px 12px;
+  width: 100%;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  list-style: none;
+}
+
+.js {
+  width: 89px !important;
+}
+
+.card button {
+  padding: 1%;
+  width: 38%;
+  height: 48px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  font-weight: 500;
+  font-size: 105%;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+}
+
+/* About me  */
+.about {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  width: 100%;
+  background: white;
+  margin-top: 180px;
+  padding-top: 180px;
+  padding-bottom: 100px;
+  display: flex;
+  justify-content: center;
+  border-top-right-radius: 150px;
+}
+
+.about > div {
+  width: 90%;
+  margin: auto;
+}
+
+/* .about > div > h2 {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 130%;
+  color: #172b4d;
+} */
```

Comment:
- [ ] Kindly remove the commented unused pieces of code from your CSS file, comments like this make the code messy and unprofessional 😓